modular_skyrat/modules/importedvendors/code/vendors.dm

```diff
@@ -0,0 +1,182 @@
+				/obj/item/reagent_containers/cup/soda_cans/pwr_game = 6,
+			),
+		),
+		list(
+			"name" = "Meals",
+			"icon" = "pizza-slice",
+			"products" = list(
+				/obj/item/storage/box/foodpack/nt = 6,
+				/obj/item/storage/box/foodpack/nt/burger = 6,
+				/obj/item/storage/box/foodpack/nt/chickensammy = 6,
+				/obj/item/food/vendor_tray_meal/side = 6,
+				/obj/item/food/vendor_tray_meal/side/crackers_and_jam = 6,
+				/obj/item/food/vendor_tray_meal/side/crackers_and_cheese = 6,
+			),
+		),
+	)
+
+	refill_canister = /obj/item/vending_refill/snack
+	default_price = PAYCHECK_CREW * 0.5
+	extra_price = PAYCHECK_COMMAND
+	payment_department = NO_FREEBIES
+
+	/// What language should this vendor speak, for flavor reasons
+	var/language_to_speak = /datum/language/common
+
+/obj/machinery/vending/imported/Initialize(mapload)
+	. = ..()
+	var/datum/language_holder/vendor_languages = get_language_holder()
+	grant_all_languages()
+	vendor_languages.selected_language = language_to_speak
+
+/* Pending me speaking to a russian (terrifying experience)
+/obj/machinery/vending/imported/panslav
+	language_to_speak = /datum/language/panslavic
+*/
+
+/obj/machinery/vending/imported/yangyu
+	name = "質の高い食品ベンダー"
+	desc = "A vendor selling traditionally sol eastern foods of dubious quality, 'don't trust the sushi' is written on the side in marker."
+	icon_state = "yangyufood"
+	light_mask = "yangyufood-light-mask"
+	light_color = LIGHT_COLOR_FLARE
+	product_slogans = "Fresh farmed space carp from local space!;Imitation lobstrocity sushi choices availible!;Made with traditional recipes and care!"
+	product_categories = list(
+		list(
+			"name" = "Snacks",
+			"icon" = "cookie",
+			"products" = list(
+				/obj/item/reagent_containers/cup/glass/dry_ramen/prepared = 6,
+				/obj/item/reagent_containers/cup/glass/dry_ramen/prepared/hell = 6,
+				/obj/item/food/vendor_snacks/ricecrackers = 6,
+				/obj/item/food/vendor_snacks/mochi_icecream = 6,
+				/obj/item/food/vendor_snacks/mochi_icecream/matcha = 6,
+				/obj/item/reagent_containers/cup/glass/waterbottle/tea = 6,
+				/obj/item/reagent_containers/cup/glass/waterbottle/tea/astra = 6,
+				/obj/item/reagent_containers/cup/glass/waterbottle/tea/strawberry = 6,
+				/obj/item/reagent_containers/cup/glass/waterbottle/tea/nip = 6,
+			),
+		),
+		list(
+			"name" = "Meals",
+			"icon" = "pizza-slice",
+			"products" = list(
+				/obj/item/storage/box/foodpack/yangyu = 6,
+				/obj/item/storage/box/foodpack/yangyu/sushi = 6,
+				/obj/item/storage/box/foodpack/yangyu/beefrice = 6,
+				/obj/item/food/vendor_tray_meal/side/miso = 6,
+				/obj/item/food/vendor_tray_meal/side/rice = 6,
+				/obj/item/food/vendor_tray_meal/side/pickled_vegetables = 6,
+			),
+		),
+	)
+
+	language_to_speak = /datum/language/yangyu
+
+/obj/machinery/vending/imported/mothic
+	name = "Nomad Fleet Ration Chit Exchange"
+	desc = "One of the nomad fleet's own ration vendors, don't mind the name engraving, this machine just takes credits."
```

Comment:
```suggestion
	desc = "One of the Nomad Fleet's own ration vendors; in spite of the name engraved into it, it's been fitted to accept credits."
```
Don't forget to capitalize names/titles! Noticed a few, namely Sol; can't blame you for it tho because we're not meant to do it in the Name but yet can/should in the Desc and aaaaaaa